samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,392 @@
+
+Configuration options
+=====================
+
+Check and configure the following configuration option for the sample:
+
+.. _CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM:
+
+CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM - Configuration for the sequence number.
+   The sample configuration updates the sequence number of the SUIT envelope, which is reflected as the version of the application in the nRF Device Manager app.
+   The default value is ``1``.
+
+Modify partition sizes
+======================
+
+You can also modify the size and location of the partitions.
+This is done by modifying the values for the desired location and size of the partition in the devicetree :file:`.overlay` files.
+
+* To modify the Application Core’s partition size,  modify the values for ``cpuapp_slot0_partition`` defined in the :file:`zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi` file within the Zephyr repository.
+
+* To modify the DFU partition, modify the values for ``dfu_partition`` defined in :file:`samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp.overlay`.
+
+    * ``dfu_partition`` is where the update candidate is stored before the update process begins.
+
+Manifest template files
+=======================
+
+All required manifest template files are automatically created after the sample is built for the first time.
+These are needed to create a SUIT envelope to carry out the DFU.
+
+.. note::
+
+   The Radio Domain manifest template is only created when building the Bluetooth Low Energy version of the sample, and not the UART version.
+   Currently, it is not needed for the UART version.
+
+If you want to make modifications to how the DFU is executed in this sample, you can do so by editing the manifest templates.
+See the :ref:`ug_nrf54h20_suit_customize_dfu` user guide for instructions and examples.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/suit/smp_transfer`
+
+This sample can be found under |sample path| in the |NCS| folder structure.
+
+Building and programming using the command line
+===============================================
+
+To build and program the sample to the nRF54H20 DK, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Over Bluetooth Low Energy
+
+      1. Open a terminal window in |sample path|.
+      #. Build the sample using the following ``west`` command, with the following Kconfig options set:
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -DFILE_SUFFIX=bt -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
+
+         .. note::
+
+            If you are compiling in Windows and the build is unsuccessful due to the maximum path length limitation, use the following command:
+
+            .. code-block:: console
+
+               west build -p -b nrf54h20dk/nrf54h20/cpuapp -d C:/ncs-lcs/work-dir --sysbuild -- -DFILE_SUFFIX=bt -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
```

Comment:
Please integrate this whole note with the shortcut we already have: |application_sample_long_path_windows| and that is used on https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/configuring_app/index.html

You can for example expand the shortcut with the mention of "for example, when building on the command line, you can add the shortened path using the ``-d`` parameter". Then, you can use the shortcut here, and keep the example build command.